### PR TITLE
Fix pivot count

### PR DESCRIPTION
**Pull request by minahlee**
Ready to merge

### CODE REVIEW

**Comment by corneadoug** [comment]
Tested the fix, working nicely

**Comment by swkimme** [comment]
LGTM!

**Comment by Leemoonsoo** [comment]
Tested and LGTM!

File: zeppelin-web/app/scripts/controllers/paragraph.js [hunk 1]
```diff
@@ -1115,7 +1115,7 @@ angular.module('zeppelinWebApp')
         // add value to row
         if (!p[valueKey]) {
           p[valueKey] = {
-              value : row[value.index],
+              value : (value.aggr != 'count') ? row[value.index] : 1,
```
Comment (anthonycorbacho) [inline zeppelin-web/app/scripts/controllers/paragraph.js]: please `!===`

**Pull request merged**